src/rf/apps/workers/sqs_manager.py

```diff
@@ -19,51 +18,51 @@ class SQSManager(object):
     Manages SQS queue creation and message functionality
     """
     def __init__(self):
-        self.q = self._get_queue()
+        self.client = boto3.client('sqs')
+        self.queue_url = self.client.get_queue_url(
+            QueueName=settings.AWS_SQS_QUEUE)['QueueUrl']
 
     def get_message(self):
-        message = self.q.read(wait_time_seconds=MAX_WAIT,
-                              message_attributes=['All'])
-        if message:
-            # Return raw message and parsed body.
-            return {
-                'message': message,
-                'body': json.loads(message.get_body())
-            }
-        else:
-            return None
+        response = self.client.receive_message(QueueUrl=self.queue_url,
+                                               WaitTimeSeconds=20)
```

Comment:
I'd like to wait until another part of the application needs to reference the same value before creating a constant.